### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Gallolouys**
Update the game over message so people know how to play again!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Gallolouys) [inline index.html:50]:
```suggestion
<h2 hidden>CACA!</h2>
```

**Review by Axel3110 — COMMENTED** [review]
Hey regarde mes comments !

**Review by Gallolouys — COMMENTED** [review]
c'est ok bg

File: README.md [hunk 2]
```diff
@@ -216,7 +216,7 @@ Next, wait about 20 seconds for actions to run, then refresh this course's READM
   Define terms and link to docs.github.com.
 -->
 
-<details id=6>
+<details id=6 open>
```
Comment (Gallolouys) [inline README.md:219]:
```suggestion
<details id=6 closed>
```

File: index.html [hunk 3]
```diff
@@ -47,8 +47,8 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (Axel3110) [inline index.html:48]:
```suggestion
</table&chaise>
```

File: README.md [hunk 4]
```diff
@@ -216,7 +216,7 @@ Next, wait about 20 seconds for actions to run, then refresh this course's READM
   Define terms and link to docs.github.com.
 -->
 
-<details id=6>
+<details id=6 closed>
```
Comment (Axel3110) [inline README.md:219]:
```suggestion
<details id=6 open>
```

**Review by Axel3110 — APPROVED** [review]

**Pull request merged**